cppcache/integration/test/AuthInitializeTest.cpp

```diff
@@ -68,7 +68,7 @@ const int32_t CQ_PLUS_AUTH_TEST_REGION_ENTRY_COUNT = 100000;
 
 Cache createCache(std::shared_ptr<SimpleAuthInitialize> auth) {
   auto cache = CacheFactory()
-                   .set("log-level", "debug")
+                   .set("log-level", "none")
```

Comment:
Since this test is explicitly setting a log file I suspect it might be slurping the logs for its test. If it is, bad but we shouldn't set to `none` here and break the test. If it isn't slurping the logs then remove the log `log-file` lines too.

Reply:
I don't think it is, will just clean this up the rest of the way.

Reply:
There was 1 or 2 others like it.

Reply:
Got em.